source/userguide/kb.rst

```diff
@@ -701,3 +701,88 @@ In the example below, we search for restaurants whose names best match ``firetra
 .. note::
 
    We can set the ``size`` parameter of the :meth:`execute()` method to specify the maximum number of records.
+
+
+Dealing with unstructured text
+------------------------------
+The knowledge bases described in the previous sections are comprised of clearly defined data types whose pattern makes them easily searchable. We call them structured data.
+
+In this section we describe how to use the Question Answerer on knowledge bases with long, free-form text or unstructured data. For example, a company's Human Resource (HR) team could have documents about general policies of the company. We can use this data to develop a HR assistant that automatically answers any policy related questions.
+
+To use the question answerer on unstructured text, we would follow the same steps mentioned before to prepare, index and load the knowledge base.
+Here is an example of what the knowledge base can look like for the HR assistant usecase. It consists of frequently asked question and answer pairs.
+
+.. code:: python
+
+	from mindmeld.components import QuestionAnswerer
+	qa = QuestionAnswerer(app_path='hr_assistant')
+	qa.load_kb(app_namespace='hr_assistant', index_name='faq_data', data_file='hr_assistant/data/faq_data.json')
+  qa.get(index='faq_data')
+
+.. code-block:: console
+
+  [
+      {
+          "question": "What if I did not receive a form W2?",
+          "answer": "W2s are mailed to home addresses of employees as of a date in mid-January each year. If you did not receive a form W2, you may access it online (beginning with 2008 W2s), similar to employee Pay Statements. Search for and click on the W-2 task. Enter the year for the W-2 you are wanting to locate and the last 4 digits of your Social Security Number. You then will have access to the W-2 in a format that can be used for filing paper versions of federal and state tax returns.",
+          "id": "hrfaq6"
+      },
+      {
+          "question": "Are employers expected to hire the less qualified over the more qualified to meet affirmative action goals?',
+          "answer": "Employers are not expected to establish any hiring practices that conflict with the principles of sound personnel management. No one should be hired unless there is a basis for believing the individual is the best-qualified candidate. In fact, affirmative action calls for the hiring of qualified people.",
+          "id": "hrfaq12"
+      },
+      ...
+  ]
+
+The difference between the two cases is only in how MindMeld handles the search query. Internally, while the ranking algorithm remains the same for both cases, the features extracted for ranking is different and is optimized to handle long text passages rather than keyword phrases.
```

Comment:
```suggestion
The difference between the two cases is only in how MindMeld handles the search query. Internally, while the ranking algorithm remains the same for both cases, the features extracted for ranking are different and are optimized to handle long text passages rather than keyword phrases.
```